cfgov/legacy/static/nemo/_/js/search.js

```diff
@@ -1,32 +1,31 @@
 
 $(function() {
-    /* this allows us to pass in HTML tags to autocomplete. Without this they get escaped 
+    /* this allows us to pass in HTML tags to autocomplete. Without this they get escaped
```

Comment:
Maybe remove extraneous whitespace changes to reduce the size of this PR?

Reply:
You can view it without whitespace changes by adding `?w=1` to the PR URL, see https://blog.github.com/2011-10-21-github-secrets/#whitespace

Does this work for you?

Reply:
Thanks @anselmbradford, I'd forgotten about that.